Automattic/wp-calypso
# Remove sidebar upsell nudges from Calypso

#### DO NOT MERGE
Requires D69690-code to be merged first.

---

Remove sidebar upsell nudges from calypso as D69690-code moves them into
the JITM infrastructure. This means that the nudges will appear in
wp-admin as well as calypso.

#### Testing instructions



1. Apply D69690-code to your sandbox and follow testing instructions there
2. Check you don't see the same message twice on the same screen, i.e. that #58613 is fixed



Related to #52368
